com.gopeed.Gopeed.yaml

```diff
@@ -0,0 +1,55 @@
+app-id: com.gopeed.Gopeed
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+rename-desktop-file: gopeed.desktop
+command: gopeed
+finish-args:
+  - '--share=ipc'
+  - '--socket=fallback-x11'
+  - '--socket=wayland'
+  - '--share=network'
+  - '--device=dri'
+  - '--talk-name=org.kde.StatusNotifierWatcher'
+  - '--talk-name=org.freedesktop.Notifications'
+  - '--filesystem=xdg-download'
+  - '--filesystem=/tmp'
```

Comment:
```suggestion
```

You don't this, you have `TMPDIR="$XDG_RUNTIME_DIR/app/$FLATPAK_ID"`

Reply:
I used  [path_provider](https://pub.dev/packages/path_provider) to get `temporary`  directory, it got was `/tmp/xxx`.